smartcontactmanager/src/main/java/com/scm/smartcontactmanager/controller/contact/request/ContactRequestDto.java

```diff
@@ -13,6 +14,10 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class ContactRequestDto {
+	
+	@Id
```

Comment:
What is the use of this annotation here?

Reply:
Resolved.